frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-details.tsx

```diff
@@ -46,12 +46,12 @@ const Details: React.FC<DetailsProps> = ({ obj }) => {
           <dl className="co-m-pane__details">
             {size && (
               <>
-                <dt>Size</dt>
-                <dd>{sizeMetrics}</dd>
+                <dt data-test-selector="details-item-label__Size">Size</dt>
+                <dd data-test-selector="details-item-value__Size">{sizeMetrics}</dd>
               </>
             )}
-            <dt>Persistent Volume Claim</dt>
-            <dd>
+            <dt data-test-selector="details-item-label__PVC">Persistent Volume Claim</dt>
+            <dd data-test-selector="details-item-value__PVC">
```

Comment:
Can this be changed to `data-test="details-item-value__PVC"` per [best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#best-practices)?
Then in `frontend/packages/integration-tests-cypress/views/snapshot.ts` it could be:
```
export namespace SnapshotDetails {
  ...
  export const pvc = cy.byTestID('details-item-value__PVC');
  ...
```
[cy.byTestID](https://github.com/openshift/console/blob/9d9cb1f9a5279cf04c4a32f060716edda9951363/frontend/packages/integration-tests-cypress/support/selectors.ts#L15)
If this ID is unique on the page, then should be ok.